Reportedly/Modules/Authentication/LoginEmail/LoginEmailInteractor.swift

```diff
@@ -10,27 +10,43 @@ import Foundation
 
 protocol LoginEmailInteractorInput: AnyObject {
     
-    // TODO: Add login logic functions
+    func login(email: String, password: String)
 }
 
 protocol LoginEmailInteractorOutput: AnyObject {
     
-    // TODO: Add login completion callbacks functions
+    func didLogin()
+    func didFailToLogin(error: ResponseError)
```

Comment:
Can we use underscore to omit argument label? 😅

Reply:
yeah sure, let me update it